exercises/5/README.md

```diff
@@ -0,0 +1,211 @@
+# Exercise #5
```

Comment:
I think this should not appear here when you rebase on the homework-repo but im not sure...

Reply:
![2021-01-15_16-48](https://user-images.githubusercontent.com/2110676/104748016-84fbe400-5751-11eb-9a54-50b0c9e3d276.png)